config/features/flags.go

```diff
@@ -12,6 +12,11 @@ var (
 		Name:  "prater",
 		Usage: "Run Prysm configured for the Prater test network",
```

Comment:
```suggestion
		Usage: "Run Prysm configured for the Prater test network",
		Aliases: []string{"prater"},
```

Why not do this and be done?

Reply:
Thanks!